compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -994,37 +994,24 @@ bool OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::CPU *cpu, TR::ILO
     * The cases that return false are placeholders that should be updated as support for more vector evaluators is added.
     */
 
-   TR::DataType ot = opcode.getVectorResultDataType();
-   TR::DataType et = ot.getVectorElementType();
+    TR::DataType ot = opcode.getVectorResultDataType();
+    TR::DataType et = ot.getVectorElementType();
+    TR::InstOpCode nativeOpcode = TR::TreeEvaluator::getNativeSIMDOpcode(opcode.getOpCodeValue(), ot, false);
```

Comment:
Fix indendation to match surrounding code.

Reply:
It already does?